release/BUILD

```diff
@@ -432,6 +432,7 @@ py_test(
     deps = [
         ":ray_release",
         bk_require("pytest"),
+        bk_require("pytest-cov"),
```

Comment:
I think one thing is that `pytest-cov` is not really part of the dependency of these tests. like when running tests in normal way, it should not be running with coverage enabled, because it might have an impact on test performance. probably not a big deal for these release unit tests, but not the best practice for general test writing.

so it probably should be injected via some other way.

one way to do it can be we have a macro that generates two build rules, one with the coverage enabled, and one without.